src/formatDate.js

```diff
@@ -50,7 +50,30 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const array = date.split(fromFormat[3]);
```

Comment:
`array` is not a good name. Rename it to something that explains what exactly it is.